mkrelease.sh

```diff
@@ -82,21 +77,9 @@ chmod 444 $2
 }
 inst $tmpdir/${name}.tar.gz	$dst/${name}.tar.gz
 inst $tmpdir/${name}.tar.bz2	$dst/${name}.tar.bz2
-inst $tmpdir/CONTRIBUTORS	$dst/CONTRIBUTORS.txt
-inst $tmpdir/COPYING		$dst/COPYING.txt
-inst $tmpdir/README		$dst/README.txt
-inst $tmpdir/CREDITS		$dst/CREDITS.txt
-inst $tmpdir/SPONSORS		$dst/SPONSORS.txt
 inst $tmpdir/ChangeLog		$dst/ChangeLog.txt
 if [ -f $tmpdir/doc/release-notes/release-$RELEASE.html ]; then
-    cat $tmpdir/doc/release-notes/release-$RELEASE.html | sed -e '
-	s/"ChangeLog"/"ChangeLog.txt"/g;
-    ' > $tmpdir/RELEASENOTES.html
     touch -r $tmpdir/doc/release-notes/release-$RELEASE.html $tmpdir/RELEASENOTES.html
```

Comment:
I am undoing PR clearance because this change request was silently ignored when the PR was cleared for merging. Please do not ignore change requests when clearing PR for merging. Otherwise, the reviewer has to _block_ the PR to make sure their requests are noticed/addressed rather than simply overlooked.

Reply:
fixed in [1002cf1](https://github.com/squid-cache/squid/pull/1340/commits/1002cf142d06e833f98252cea85045d3b657fc7a)